### ISSUE TITLE
Only see app and etc directory

### ISSUE THREAD

**Issue by willboudle** [paliarush/vagrant-magento#65]
I'm on windows 10 and get all the way through the installation, but when I launch in browser I just see the app directory and in that only the etc directory. Any idea what I could be doing wrong?

**Comment by paliarush** [comment]
@Willboudle maybe your issue is described in [FAQ](https://github.com/paliarush/magento2-vagrant-for-developers#faq). Unfortunately, I don't have access to Windows 10 machine to reproduce it. 

@BaDos do you know what can be the issue?

**Comment by BaDos** [comment]
@paliarush no, I don't know.
It is a very strange issue.

**Comment by BaDos** [comment]
@Willboudle do you see other files of magento on windows?
If you connect to VM by ssh, do you see other files in folder /var/www/magento2ce?

**Comment by kartguru** [comment]
just FYI. I installed the on a windows 10 machine last Saturday (less  PHPStorm) . it did take me a few attempts to ensure all the necessary programs were installed ... researching all errors etc. I didn't have a requirement to follow Issue #36 though I have installed VS in a previous vagrant attempt (different build). I thought it was an earlier build though.

i have had issues, which I'm trying to work out. they relate to installation of extensions (via composer) and rendering of js components. theme and blog installed ok.
